data/card_catalogue/card-manipulation.json

```diff
@@ -4,77 +4,101 @@
     "upgrade": "Brainstorm",
     "tier": "blue",
     "details": {
-      "rules":"Cast two random Blue Cards.",
+      "rules": {
+        "active":"Draw 2 blue cards. Cast both.",
```

Comment:
I changed this set from 
> Cast two random `tier` cards

to

> Draw 2 `tier` cards. Cast both.

to specify that they're not from your hand